facp.md

```diff
@@ -0,0 +1,73 @@
+# FACP
+
+The FACP, which is also called FADT, refers to the Fixed ACPI Description Table, and is pointed by the RSDT.
```

Comment:
i would say here add soemthing like, who is calling it FADT, i know it but maybe the reader doesn't know about it so we should try to provide as more information as we can. 

Where the keyword is stored? Try to add little bit more information. Plus you can add inline links to other paragraphs too. So you could add  link to the RSDT paragraph with the following syntax: 

[RSDT](RSDP_and_RSDT.md)

Reply:
so another thing to add to this page, take note @AhnJihwan ! :)